types/mdast-util-toc-tests.ts

```diff
@@ -0,0 +1,80 @@
+import {Parent} from 'unist'
+
+import unified = require('unified')
+import u = require('unist-builder')
+import is = require('unist-util-is')
+import toc = require('mdast-util-toc')
+
+interface Link extends Parent {
+  type: 'link'
+}
+
+interface Paragraph extends Parent {
+  type: 'paragraph'
+}
+
+interface List extends Parent {
+  type: 'list'
+}
+
+interface ListItem extends Parent {
+  type: 'listItem'
+}
```

Comment:
It would be good to pull these from [`@types/mdast`](https://www.npmjs.com/package/@types/mdast), if possible.